game/webcam.cc

```diff
@@ -84,7 +84,8 @@ void Webcam::operator()() {
 				// Copy the frame to storage
 				m_frame.width = frame.cols;
 				m_frame.height = frame.rows;
-				m_frame.data.assign(frame.data, frame.data + (m_frame.width * m_frame.height * 3));
+				auto frameDims = m_frame.width * m_frame.height * 3;
+				m_frame.data.assign(frame.data, frame.data + (frameDims));
```

Comment:
Here, we could probably do better since we're using signed types for width and height. I don't know, however, whether we're dealing with OpenCV api there or not.